lib/Doctrine/ORM/Mapping/Exporter/VariableExporter.php

```diff
@@ -6,8 +6,6 @@
 
 class VariableExporter implements Exporter
 {
-    public const INDENTATION =  '    ';
```

Comment:
This is a BC break that needs documenting

Reply:
I'm doing this.

Reply:
Nope? It was causing fatal error and Exporter is 3.x only code. :blush:

Reply:
Ah... I documented this. May need removal.